src/Standards/Squiz/Sniffs/WhiteSpace/OperatorSpacingSniff.php

```diff
@@ -126,12 +129,25 @@ public function register()
      * @param int                         $stackPtr  The position of the current token in
      *                                               the stack passed in $tokens.
      *
-     * @return void
+     * @return void|int Optionally returns a stack pointer. The sniff will not be
+     *                   called again on the current file until the returned stack
+     *                   pointer is reached. Return `$phpcsFile->numTokens` to skip
+     *                   the rest of the file.
```

Comment:
When I pasted this in, my editor prefixed the lines with `* `, so it ended up with `* *     [...] called again on the [...]`. I removed the `*`, but it seems I missed the extra space. This was not identified when I ran `phpcs` over the codebase (which I do before committing). Perhaps there's a bug / oversight in the rule-set?

Reply:
I seem to remember working on a fix for this years ago and that there was some discussion on what the fix should look like... Pff.. I probably still have that branch somewhere, though it may have gotten lost on an old laptop (I have a feeling I worked on this while travelling).

Let's park it for when the docs sniff get revisited in their entirety.